src/crud.js

```diff
@@ -99,31 +101,37 @@ const showToDos = () => {
       const check = document.createElement('input');
       check.type = 'checkbox';
       check.classList.add('check');
+      check.id = `check-${index}`;
 
       const input = document.createElement('input');
       input.type = 'text';
       input.classList.add('activity');
       input.id = `activity-${index}`;
       input.setAttribute('readonly', true);
       input.value = `${todo.description}`;
-      input.innerHTML = todo.description;
 
       const edit = document.createElement('img');
       edit.setAttribute('src', dots);
       edit.classList.add('edit');
       edit.id = index;
 
       collection.appendChild(unordered);
-      collection.appendChild(liner);
       unordered.appendChild(ordered);
+      unordered.appendChild(liner);
       ordered.appendChild(check);
       ordered.appendChild(input);
       ordered.appendChild(edit);
     });
 
+    document.querySelector('#clear-completed').style.display = 'block';
+
     document.querySelectorAll('.edit').forEach((e) => {
       e.addEventListener('click', editTodos);
     });
+
+    document.querySelectorAll('.check').forEach((e) => {
+      e.addEventListener('change', completedTodos);
+    });
```

Comment:
- [ ]  Your tasks list is not keeping the localStorage state after a page refresh. Completed tasks are not checked anymore meanwhile their status is true. You should probably check your getList function to fix that.

![onchange2](https://user-images.githubusercontent.com/40503704/184208154-9a851752-ff7c-41c1-86ef-d7cac691d6f9.PNG)